docs/conops.md

```diff
@@ -0,0 +1,549 @@
+
+</center> 
+
+### 3.3 External Interfaces
+This section describes the interfaces of the system with any other systems that are external to the project. Interfaces *between* services are not shown in these diagrams.
+
+#### PSU Interfaces
+
+<center> 
+
+```mermaid
+flowchart TD
+
+subgraph Provider of Services
+	sched[svc-scheduler]
+	guid[svc-guidance]
+	dcvr[svc-discovery]
+	com[svc-compliance]
+end
+
+port[Vertiport Operator]
+
+faa{Regulatory Body} <--> com
+psus{Other PSUs} <--> dcvr
+guid --> pilot(UAM Pilot)
+sched <--> pilot
+sched <--> port
+```
+
+</center> 
+
+`svc-scheduler`:
+- Reports schedules to Vertiport Operators and Pilots in Control (PIC).
+- Vertiport Operators and Pilots may make requests to the scheduler to delay or accelerate a departure, cancel a flight, or initiate emergency operations.
+
+`svc-compliance`:
+- Information exchanged with civil aviation authorities may include notifications, confirmation of flight authorization, flight restrictions, charted routes, and Special Use/Activity Airspaces (SAA).
+- FAA-Industry Data Exchange Protocol (FIDXP) is an interface that allows UAM operators to obtain information from the United States [National Airspace System (NAS) Data Exchange](https://www.faa.gov/air_traffic/nas/).
+- Data exchange formats may differ based on region.
+
+`svc-discovery`:
+- Other PSUs may need to coordinate with this PSU in scheduling flights to vertiports.
+- Example: proprietary PSU schedules a flight from a private vertiport to a public vertiport managed by this PSU.
+- Example: regional instances of *svc-scheduler* coordinate to transfer stewardship of an aircraft departing one PSU's region into another PSU's region.
+
+`svc-guidance`:
+- Calculating flight paths will be increasingly important as the number of simultaneous airborne UAM vehicles increases.
+- Flight paths are chosen at the discretion of the Pilot in Control (PIC). Suggested flight paths from this service should be taken into consideration when planning the general route.
+
+#### SDSP Interfaces
+
+<center> 
+
+```mermaid
+
+flowchart BT
+subgraph Supplemental Data and Service Providers
+	stor[svc-storage]
+	tlm[svc-telemetry]
+	upk[svc-upkeep]
+end
+
+gcs{Ground Control/<br>Air Traffic Control<br>Interfaces}
+vehicle[UAM Vehicle] -- Telemetry --> tlm
+stor <--> cloud{Cloud Storage}
+tlm --> gcs
+
+```
+
+</center> 
+
+`svc-telemetry`:
+- UAM vehicles broadcast telemetry, which is processed and stored by this service.
+- This service may re-broadcast telemetry to clients such as Air Traffic Control (ATC) software.
+
+`svc-storage`:
+- Links to multiple storage solutions, paired with specific types of data.
+- Blockchain storage provides defensive "immutable" record keeping for certifications, maintenance records, previous flight plans, FAA confirmations.
+- SQL databases for CRUD operations on flight plans
```

Comment:
Have we decided to go forward with SQL? I think we should do some research on what types of db is suitable for flight plans.

Reply:
Will remove 'SQL', too specific anyway

Reply:
https://github.com/Arrow-air/se-services/pull/2/commits/fb227c5a17a327041869503558fcb729308c9955